tests/autoyast/installation.pm

```diff
@@ -52,10 +52,8 @@ sub save_and_upload_yastlogs {
     my $name = $stage . ($suffix // '');
```

Comment:
I think the function `save_and_upload_yastlogs` needs another name. It is not doing that anymore and it is confusing.

Reply:
I've missed something here, so thanks a lot for noticing this